hudi-hadoop-mr/src/main/java/org/apache/hudi/hadoop/hive/HoodieCombineHiveInputFormat.java

```diff
@@ -183,8 +183,8 @@ protected HoodieCombineFileInputFormatShim createInputFormatShim() {
         deserializerClassName = part.getDeserializer(job).getClass().getName();
       } catch (Exception e) {
         // ignore
+        LOG.error("can not get deserializerClassName, occur exception ", e);
       }
```

Comment:
LOG.error("Getting deserializer class name error",  e);

Reply:
ok i updated it, but ci/cd report checkstyle error, not know the reason.